src/ui/components/DownloadsManagerView/index.tsx

```
@@ -237,15 +238,15 @@ const DownloadsManagerView = () => {
             onChange={handleServerFilterChange}
           />
         </Box>
-        <Box display='flex' flexGrow={3} flexShrink={3} paddingInline={2}>
+        <Box display='flex' flexGrow={3} flexShrink={3} paddingInline={2} style={{ color: '#C0C0C0' }}>
```

ESLint

[error] Replace `·display='flex'·flexGrow={3}·flexShrink={3}·paddingInline={2}·style={{·color:·'#C0C0C0'·}}` with `⏎··········display='flex'⏎··········flexGrow={3}⏎··········flexShrink={3}⏎··········paddingInline={2}⏎··········style={{·color:·'#C0C0C0'·}}⏎········`

(prettier/prettier)